src/EnergyPlus/ChillerIndirectAbsorption.cc

```diff
@@ -1559,6 +1564,19 @@ namespace ChillerIndirectAbsorption {
 		LoopSideNum = IndirectAbsorber( ChillNum ).CWLoopSideNum;
 
 		CpFluid = GetSpecificHeatGlycol( PlantLoop( IndirectAbsorber( ChillNum ).CWLoopNum ).FluidName, EvapInletTemp, PlantLoop( IndirectAbsorber( ChillNum ).CWLoopNum ).FluidIndex, RoutineName );
+		
+		//If there is a fault of Chiller SWT Sensor (zrp_Jun2016)
+		if( IndirectAbsorber( ChillNum ).FaultyChillerSWTFlag && ( ! WarmupFlag ) && ( ! DoingSizing ) && DoWeathSim ){
```

Comment:
I guess I was thinking DoWeathSim should either go away completely, or be replaced by something like 
       (  ( DataGloabals::KindOfSim == ksRunPeriodWeather ) || ( DataGloabals::KindOfSim == ksDesignDay ) )

It can also be good to exclude kick off timesteps using 

       ! DataGlobals::KickOffSimulation